usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -25,6 +25,8 @@
 
 from usaspending_api.etl.management.commands.create_delta_table import TABLE_SPEC
 
+DEFAULT_TEXT_SEARCH_CONFIG = "pg_catalog.simple"
```

Comment:
While this is used here, it feels like it should get brought up to a higher level. Like `settings.py` ... or if it's only used in spark stuff for now, possibly in `usaspending_api.config.envs.default.py`

Reply:
It's just the default setting of our Postgres DBs - so ill just move it to settings.py.